SU-SWS/stanford_profile
# D8CORE-6288 Policy content type

# READY FOR REVIEW

# Summary
- Policy content type
- Works with https://github.com/SU-SWS/stanford_fields/pull/35

# Need Review By (Date)
- ASAP

# Urgency
- high

# Steps to Test
1. `composer require su-sws/stanford_profile:dev-D8CORE-6288`
2. install new site or import configs
3. create a new "Policy" node
   1. The first node will be the "Book" that will bind all the policies togeather
   2. In the right sidebar, choose `- Create a new book -`
   3. Populate other fields as desired
   4. Save the node
4. Repeat the above step but choose the "Book" you created above.
5. Choose it's parent item and it's position similar to adding a page to the main menu.
6. Repeat with various parent levels and different orders
7. Verify the left sub navigation changes and matches what is expected on each save.
8. Populate some of the change logs and verify they show on the bottom of the page
9. Go to the config page at `/admin/config/content/policy`
10. change the prefix settings
11. review the changes on the node pages to verify your choices have been reflected.
